elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -10,10 +12,10 @@ $pfe-cta__arrow-size:                        0.313em;
 /// DEFAULT CTA
 /// ===========================================================================
 :host {
-  --pfe-cta--main:                     #{pfe-color(ui-link)};
-  --pfe-cta--main--hover:              #{pfe-color(ui-link--hover)};
-  --pfe-cta--main--focus:              #{pfe-color(ui-link--focus)};
-  --pfe-cta--main--visited:            #{pfe-color(ui-link--visited)};
+  --pfe-cta--main:                     #{pfe-apply-broadcast(ui-link)};
```

Comment:
Utilizing broadcast values here for initials

Reply:
Just noodling on naming / docs to see if I'm understanding it correctly, tell me if I'm off :)

1. `@mixin pfe-set-broadcast` i.e. Set initial values for list of the broadcast variables: `$BROADCAST-VARS`
2. `@function pfe-apply-broadcast` i.e. Make use of a broadcast variable as a value for the color property. Possible naming suggestions:
    - `@function pfe-invoke`
    - `@function pfe-use-broadcast`
    - `@function pfe-broadcast-color`